pkg/util/limiter/errors.go

```diff
@@ -30,18 +32,24 @@ var (
 	)
 )
 
+func limitErrorFunc(format string) func(uint64) validation.LimitError {
+	return func(limit uint64) validation.LimitError {
+		return validation.LimitError(fmt.Sprintf(format, limit))
+	}
+}
```

Comment:
and at this point can't `limitErrorFunc` become  `limitError`
```go
func limitError(format string, limit uint64) validation.LimitError {
	return validation.LimitError(fmt.Sprintf(format, limit))
}
```

Reply:
You are right, it can. Will fix it right now